<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>test report</title>
    <meta name="generator" content="HTMLTestRunner 0.8.2"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
	<link rel="stylesheet" href="http://cdn.bootcss.com/bootstrap/3.3.0/css/bootstrap.min.css">
	<script src="http://cdn.bootcss.com/bootstrap/3.3.0/js/bootstrap.min.js"></script>
    
<style type="text/css" media="screen">
body        { font-family: verdana, arial, helvetica, sans-serif; font-size: 80%; }
table       { font-size: 100%; }
pre         { }

/* -- heading ---------------------------------------------------------------------- */
h1 {
	font-size: 16pt;
	color: gray;
}
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}

.heading .attribute {
    margin-top: 1ex;
    margin-bottom: 0;
}

.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}

/* -- css div popup ------------------------------------------------------------------------ */
a.popup_link {
}

a.popup_link:hover {
    color: red;
}

.popup_window {
    display: none;
    position: relative;
    left: 0px;
    top: 0px;
    /*border: solid #627173 1px; */
    padding: 10px;
    background-color: #E6E6D6;
    font-family: "Lucida Console", "Courier New", Courier, monospace;
    text-align: left;
    font-size: 8pt;
    width: 500px;
}

}
/* -- report ------------------------------------------------------------------------ */
#show_detail_line {
    margin-top: 3ex;
    margin-bottom: 1ex;
}
#result_table {
    width: 80%;
    border-collapse: collapse;
    border: 1px solid #777;
}
#header_row {
    font-weight: bold;
    color: white;
    background-color: #777;
}
#result_table td {
    border: 1px solid #777;
    padding: 2px;
}
#total_row  { font-weight: bold; }
.passClass  { background-color: #6c6; }
.failClass  { background-color: #c60; }
.errorClass { background-color: #c00; }
.passCase   { color: #6c6; }
.failCase   { color: #c60; font-weight: bold; }
.errorCase  { color: #c00; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }


/* -- ending ---------------------------------------------------------------------- */
#ending {
}

</style>

</head>
<body>
<script language="javascript" type="text/javascript"><!--
output_list = Array();

/* level - 0:Summary; 1:Failed; 2:All */
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level < 1) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level > 1) {
                tr.className = '';
            }
            else {
                tr.className = 'hiddenRow';
            }
        }
    }
}


function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        tid0 = 't' + cid.substr(1) + '.' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        if (toHide) {
            document.getElementById('div_'+tid).style.display = 'none'
            document.getElementById(tid).className = 'hiddenRow';
        }
        else {
            document.getElementById(tid).className = '';
        }
    }
}


function showTestDetail(div_id){
    var details_div = document.getElementById(div_id)
    var displayState = details_div.style.display
    // alert(displayState)
    if (displayState != 'block' ) {
        displayState = 'block'
        details_div.style.display = 'block'
    }
    else {
        details_div.style.display = 'none'
    }
}


function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}

/* obsoleted by detail in <div>
function showOutput(id, name) {
    var w = window.open("", //url
                    name,
                    "resizable,scrollbars,status,width=800,height=450");
    d = w.document;
    d.write("<pre>");
    d.write(html_escape(output_list[id]));
    d.write("\n");
    d.write("<a href='javascript:window.close()'>close</a>\n");
    d.write("</pre>\n");
    d.close();
}
*/
--></script>

<div class='heading'>
<h1>test report</h1>
<p class='attribute'><strong>Start Time:</strong> 2018-05-17 09:55:04</p>
<p class='attribute'><strong>Duration:</strong> 0:00:32.099836</p>
<p class='attribute'><strong>Status:</strong> Error 33</p>

<p class='description'>apd_test market manager</p>
</div>



<p id='show_detail_line'>Show
<a href='javascript:showCase(0)' class="btn btn-xs btn-primary">Summary</a>
<a href='javascript:showCase(1)' class="btn btn-xs btn-danger">Failed</a>
<a href='javascript:showCase(2)' class="btn btn-xs btn-info">All</a>
</p>
<table id='result_table'>
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row'>
    <td>Test Group/Test case</td>
    <td>Count</td>
    <td>Pass</td>
    <td>Fail</td>
    <td>Error</td>
    <td>View</td>
</tr>

<tr class='errorClass'>
    <td>unittest.loader._FailedTest</td>
    <td>25</td>
    <td>0</td>
    <td>0</td>
    <td>25</td>
    <td><a href="javascript:showClassDetail('c1',25)">Detail</a></td>
</tr>

<tr id='ft1.1' class='none'>
    <td class='errorCase'><div class='testcase'>borrowtips_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.1')" >
        error</a>

    <div id='div_ft1.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.1: ImportError: Failed to import test module: test_cases.borrowtips_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\borrowtips_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.2' class='none'>
    <td class='errorCase'><div class='testcase'>carborrow_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.2')" >
        error</a>

    <div id='div_ft1.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.2: ImportError: Failed to import test module: test_cases.carborrow_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\carborrow_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.3' class='none'>
    <td class='errorCase'><div class='testcase'>chargestandard_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.3')" >
        error</a>

    <div id='div_ft1.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.3: ImportError: Failed to import test module: test_cases.chargestandard_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\chargestandard_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.4' class='none'>
    <td class='errorCase'><div class='testcase'>companyhonor_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.4')" >
        error</a>

    <div id='div_ft1.4' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.4').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.4: ImportError: Failed to import test module: test_cases.companyhonor_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\companyhonor_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.5' class='none'>
    <td class='errorCase'><div class='testcase'>companyintro_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.5')" >
        error</a>

    <div id='div_ft1.5' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.5').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.5: ImportError: Failed to import test module: test_cases.companyintro_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\companyintro_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.6' class='none'>
    <td class='errorCase'><div class='testcase'>companynews_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.6')" >
        error</a>

    <div id='div_ft1.6' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.6').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.6: ImportError: Failed to import test module: test_cases.companynews_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\companynews_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.7' class='none'>
    <td class='errorCase'><div class='testcase'>companyteam_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.7')" >
        error</a>

    <div id='div_ft1.7' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.7').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.7: ImportError: Failed to import test module: test_cases.companyteam_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\companyteam_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.8' class='none'>
    <td class='errorCase'><div class='testcase'>cooperation_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.8')" >
        error</a>

    <div id='div_ft1.8' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.8').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.8: ImportError: Failed to import test module: test_cases.cooperation_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\cooperation_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.9' class='none'>
    <td class='errorCase'><div class='testcase'>helpcenter_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.9')" >
        error</a>

    <div id='div_ft1.9' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.9').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.9: ImportError: Failed to import test module: test_cases.helpcenter_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\helpcenter_test.py", line 8, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.10' class='none'>
    <td class='errorCase'><div class='testcase'>lawsothers_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.10')" >
        error</a>

    <div id='div_ft1.10' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.10').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.10: ImportError: Failed to import test module: test_cases.lawsothers_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\lawsothers_test.py", line 8, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.11' class='none'>
    <td class='errorCase'><div class='testcase'>lendertips_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.11')" >
        error</a>

    <div id='div_ft1.11' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.11').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.11: ImportError: Failed to import test module: test_cases.lendertips_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\lendertips_test.py", line 8, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.12' class='none'>
    <td class='errorCase'><div class='testcase'>lows_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.12')" >
        error</a>

    <div id='div_ft1.12' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.12').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.12: ImportError: Failed to import test module: test_cases.lows_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\lows_test.py", line 8, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.13' class='none'>
    <td class='errorCase'><div class='testcase'>mainpublish_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.13')" >
        error</a>

    <div id='div_ft1.13' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.13').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.13: ImportError: Failed to import test module: test_cases.mainpublish_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\mainpublish_test.py", line 8, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.14' class='none'>
    <td class='errorCase'><div class='testcase'>newsreport_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.14')" >
        error</a>

    <div id='div_ft1.14' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.14').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.14: ImportError: Failed to import test module: test_cases.newsreport_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\newsreport_test.py", line 8, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.15' class='none'>
    <td class='errorCase'><div class='testcase'>operationalreport_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.15')" >
        error</a>

    <div id='div_ft1.15' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.15').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.15: ImportError: Failed to import test module: test_cases.operationalreport_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\operationalreport_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.16' class='none'>
    <td class='errorCase'><div class='testcase'>p2pknowledge_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.16')" >
        error</a>

    <div id='div_ft1.16' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.16').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.16: ImportError: Failed to import test module: test_cases.p2pknowledge_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\p2pknowledge_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.17' class='none'>
    <td class='errorCase'><div class='testcase'>plateformdate_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.17')" >
        error</a>

    <div id='div_ft1.17' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.17').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.17: ImportError: Failed to import test module: test_cases.plateformdate_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\plateformdate_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.18' class='none'>
    <td class='errorCase'><div class='testcase'>plateformnotice_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.18')" >
        error</a>

    <div id='div_ft1.18' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.18').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.18: ImportError: Failed to import test module: test_cases.plateformnotice_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\plateformnotice_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.19' class='none'>
    <td class='errorCase'><div class='testcase'>position_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.19')" >
        error</a>

    <div id='div_ft1.19' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.19').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.19: ImportError: Failed to import test module: test_cases.position_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\position_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.20' class='none'>
    <td class='errorCase'><div class='testcase'>product_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.20')" >
        error</a>

    <div id='div_ft1.20' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.20').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.20: ImportError: Failed to import test module: test_cases.product_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\product_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.21' class='none'>
    <td class='errorCase'><div class='testcase'>prohibitedbv_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.21')" >
        error</a>

    <div id='div_ft1.21' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.21').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.21: ImportError: Failed to import test module: test_cases.prohibitedbv_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\prohibitedbv_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.22' class='none'>
    <td class='errorCase'><div class='testcase'>regulators_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.22')" >
        error</a>

    <div id='div_ft1.22' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.22').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.22: ImportError: Failed to import test module: test_cases.regulators_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\regulators_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.23' class='none'>
    <td class='errorCase'><div class='testcase'>riskeducation_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.23')" >
        error</a>

    <div id='div_ft1.23' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.23').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.23: ImportError: Failed to import test module: test_cases.riskeducation_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\riskeducation_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.24' class='none'>
    <td class='errorCase'><div class='testcase'>riskmanagement_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.24')" >
        error</a>

    <div id='div_ft1.24' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.24').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.24: ImportError: Failed to import test module: test_cases.riskmanagement_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\riskmanagement_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft1.25' class='none'>
    <td class='errorCase'><div class='testcase'>safety_test</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft1.25')" >
        error</a>

    <div id='div_ft1.25' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft1.25').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft1.25: ImportError: Failed to import test module: test_cases.safety_test
Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 298, in discover
    os.path.dirname((the_module.__file__)))
AttributeError: module 'test_cases' has no attribute '__file__'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\python3.6\lib\unittest\loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "C:\python3.6\lib\unittest\loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "F:/python01/UI_test/market\test_cases\safety_test.py", line 9, in &lt;module&gt;
    from driver import my_driver
ImportError: cannot import name 'my_driver'



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>test_cases.creditborrow_test.CreditborrowTest</td>
    <td>3</td>
    <td>0</td>
    <td>0</td>
    <td>3</td>
    <td><a href="javascript:showClassDetail('c2',3)">Detail</a></td>
</tr>

<tr id='ft2.1' class='none'>
    <td class='errorCase'><div class='testcase'>test1_creditbrorrow_success: 信用借款信息请求提交成功</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.1')" >
        error</a>

    <div id='div_ft2.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft2.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft2.1: Traceback (most recent call last):
  File "F:/python01/UI_test/market\test_cases\creditborrow_test.py", line 23, in test1_creditbrorrow_success
    self.creditborrow_p.form_submit()
  File "F:/python01/UI_test/market/pages\creditborrow.py", line 43, in form_submit
    self.username.send_keys(username)
  File "F:/python01/UI_test/market/pages\creditborrow.py", line 10, in username
    return self.by_id("lendName")
  File "F:/python01/UI_test/market/pages\basepage.py", line 22, in by_id
    return self.driver.find_element_by_id(id)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 229, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 707, in find_element
    {'using': by, 'value': value})['value']
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 196, in execute
    self.error_handler.check_response(response)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 181, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=49.0.2623.13)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft2.2' class='none'>
    <td class='errorCase'><div class='testcase'>test2_codeinput_error: 验证码输入错误，后台验证</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.2')" >
        error</a>

    <div id='div_ft2.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft2.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft2.2: Traceback (most recent call last):
  File "F:/python01/UI_test/market\test_cases\creditborrow_test.py", line 29, in test2_codeinput_error
    self.creditborrow_p.form_submit()
  File "F:/python01/UI_test/market/pages\creditborrow.py", line 43, in form_submit
    self.username.send_keys(username)
  File "F:/python01/UI_test/market/pages\creditborrow.py", line 10, in username
    return self.by_id("lendName")
  File "F:/python01/UI_test/market/pages\basepage.py", line 22, in by_id
    return self.driver.find_element_by_id(id)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 229, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 707, in find_element
    {'using': by, 'value': value})['value']
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 196, in execute
    self.error_handler.check_response(response)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 181, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=49.0.2623.13)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft2.3' class='none'>
    <td class='errorCase'><div class='testcase'>test3_allinput_error: 所有输入均为空</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft2.3')" >
        error</a>

    <div id='div_ft2.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft2.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft2.3: Traceback (most recent call last):
  File "F:/python01/UI_test/market\test_cases\creditborrow_test.py", line 35, in test3_allinput_error
    self.creditborrow_p.btn.click()
  File "F:/python01/UI_test/market/pages\creditborrow.py", line 38, in btn
    return self.by_tag_name("button")
  File "F:/python01/UI_test/market/pages\basepage.py", line 43, in by_tag_name
    return self.driver.find_element_by_tag_name(tagname)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 349, in find_element_by_tag_name
    return self.find_element(by=By.TAG_NAME, value=name)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 707, in find_element
    {'using': by, 'value': value})['value']
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 196, in execute
    self.error_handler.check_response(response)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 181, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=49.0.2623.13)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>unittest.suite._ErrorHolder</td>
    <td>2</td>
    <td>0</td>
    <td>0</td>
    <td>2</td>
    <td><a href="javascript:showClassDetail('c3',2)">Detail</a></td>
</tr>

<tr id='ft3.1' class='none'>
    <td class='errorCase'><div class='testcase'>HomePageTest)</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft3.1')" >
        error</a>

    <div id='div_ft3.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft3.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft3.1: Traceback (most recent call last):
  File "F:/python01/UI_test/market\test_cases\home_page_test.py", line 18, in setUpClass
    self.hp.layerclose()
  File "F:/python01/UI_test/market/pages\homepage.py", line 12, in layerclose
    return self.by_css("div.layerbg&gt;div.layerclose").click()
  File "F:/python01/UI_test/market/pages\basepage.py", line 28, in by_css
    return self.driver.find_element_by_css_selector(css_loc)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 397, in find_element_by_css_selector
    return self.find_element(by=By.CSS_SELECTOR, value=css_selector)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 707, in find_element
    {'using': by, 'value': value})['value']
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 196, in execute
    self.error_handler.check_response(response)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 181, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=49.0.2623.13)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft3.2' class='none'>
    <td class='errorCase'><div class='testcase'>InvestTest)</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft3.2')" >
        error</a>

    <div id='div_ft3.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft3.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft3.2: Traceback (most recent call last):
  File "F:/python01/UI_test/market\test_cases\invest_test.py", line 20, in setUpClass
    self.lg.login(username='13658524695', password='123456')
  File "F:/python01/UI_test/market/pages\loginpage.py", line 44, in login
    self.input_userName().send_keys(username)
  File "F:/python01/UI_test/market/pages\loginpage.py", line 12, in input_userName
    return self.by_id("mobile")
  File "F:/python01/UI_test/market/pages\basepage.py", line 22, in by_id
    return self.driver.find_element_by_id(id)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 229, in find_element_by_id
    return self.find_element(by=By.ID, value=id_)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 707, in find_element
    {'using': by, 'value': value})['value']
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 196, in execute
    self.error_handler.check_response(response)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 181, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=49.0.2623.13)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr class='errorClass'>
    <td>test_cases.login_test.LoginTest: 测试前台登录</td>
    <td>3</td>
    <td>0</td>
    <td>0</td>
    <td>3</td>
    <td><a href="javascript:showClassDetail('c4',3)">Detail</a></td>
</tr>

<tr id='ft4.1' class='none'>
    <td class='errorCase'><div class='testcase'>test1_username_error</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.1')" >
        error</a>

    <div id='div_ft4.1' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.1').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.1: Traceback (most recent call last):
  File "F:/python01/UI_test/market\test_cases\login_test.py", line 22, in test1_username_error
    self.lg.login(username='11111', password='123456')
  File "F:/python01/UI_test/market/pages\loginpage.py", line 43, in login
    self.open()
  File "F:/python01/UI_test/market/pages\basepage.py", line 19, in open
    self._open(self.url)
  File "F:/python01/UI_test/market/pages\basepage.py", line 16, in _open
    self.driver.get(url)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 208, in get
    self.execute(Command.GET, {'url': url})
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 196, in execute
    self.error_handler.check_response(response)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 181, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=49.0.2623.13)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.2' class='none'>
    <td class='errorCase'><div class='testcase'>test2_password_error</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.2')" >
        error</a>

    <div id='div_ft4.2' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.2').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.2: Traceback (most recent call last):
  File "F:/python01/UI_test/market\test_cases\login_test.py", line 29, in test2_password_error
    self.lg.login(username='13658524695', password='111111')
  File "F:/python01/UI_test/market/pages\loginpage.py", line 43, in login
    self.open()
  File "F:/python01/UI_test/market/pages\basepage.py", line 19, in open
    self._open(self.url)
  File "F:/python01/UI_test/market/pages\basepage.py", line 16, in _open
    self.driver.get(url)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 208, in get
    self.execute(Command.GET, {'url': url})
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 196, in execute
    self.error_handler.check_response(response)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 181, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=49.0.2623.13)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='ft4.3' class='none'>
    <td class='errorCase'><div class='testcase'>test3_login_success</div></td>
    <td colspan='5' align='center'>

    <!--css div popup start-->
    <a class="popup_link" onfocus='this.blur();' href="javascript:showTestDetail('div_ft4.3')" >
        error</a>

    <div id='div_ft4.3' class="popup_window">
        <div style='text-align: right; color:red;cursor:pointer'>
        <a onfocus='this.blur();' onclick="document.getElementById('div_ft4.3').style.display = 'none' " >
           [x]</a>
        </div>
        <pre>
        
ft4.3: Traceback (most recent call last):
  File "F:/python01/UI_test/market\test_cases\login_test.py", line 37, in test3_login_success
    self.lg.login(username='13658524695', password='123456')
  File "F:/python01/UI_test/market/pages\loginpage.py", line 43, in login
    self.open()
  File "F:/python01/UI_test/market/pages\basepage.py", line 19, in open
    self._open(self.url)
  File "F:/python01/UI_test/market/pages\basepage.py", line 16, in _open
    self.driver.get(url)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 208, in get
    self.execute(Command.GET, {'url': url})
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 196, in execute
    self.error_handler.check_response(response)
  File "C:\python3.6\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 181, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=49.0.2623.13)
  (Driver info: chromedriver=2.21.371459 (36d3d07f660ff2bc1bf28a75d1cdabed0983e7c4),platform=Windows NT 6.1 SP1 x86_64)



        </pre>
    </div>
    <!--css div popup end-->

    </td>
</tr>

<tr id='total_row'>
    <td>Total</td>
    <td>33</td>
    <td class="text text-success">0</td>
    <td class="text text-danger">0</td>
    <td class="text text-warning">33</td>
    <td>&nbsp;</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>

</body>
</html>
